test/e2e/geojson/geojson2mapml.html

```diff
@@ -0,0 +1,715 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>geoJSON 2 MapML</title>
+  <meta charset="UTF-8">
+  <script type="module" src="mapml-viewer.js"></script>
+  <script src="lib/geojson.js"></script>
+</head>
+
+<body>
+    <mapml-viewer id="output" projection="OSMTILE" zoom="10" lon="-75.7" lat="45.4" controls>
+        <!--geojson2mapml layer output goes here dynamically-->
+    </mapml-viewer>
+
+    <!-- The expected results for geojson2mapml go here-->
+    <mapml-viewer id="expected1" projection="OSMTILE" zoom="10" lon="-75.7" lat="45.4" controls>

```

Comment:
It may be the case that the many mapml-viewers here aren't necessary, since the actual layers never get rendered, they're just used for comparison with the output of geojson2mapml, perhaps it would be faster to just put the id on the `<layer->` elements and compare them directly.  You can always put a `hidden` attribute on the layer to prevent it from being displayed with a FOUC, I believe.  Anyway, not a big deal because the tests work as is.

Reply:
Good point, removed mapml-viewers.